app/src/pages/PageProjects.re

```diff
@@ -52,9 +48,13 @@ let make = () => {
   });
 
   <>
-    <div>
-      <Title.Huge> {React.string("Explore")} </Title.Huge>
-      <Button> {React.string("Register project")} </Button>
+    <div className=Styles.projectHeading>
+      <Container.TwoColumns>
+        ...(
```

Comment:
What does this spread do here?

Reply:
Same as [above](https://github.com/oscoin/mvp/pull/10/files#r315655987).